treeherder/etl/perf.py

```diff
@@ -100,68 +100,41 @@ def load_perf_artifacts(project_name, reference_data, job_data, datum):
 
     framework = PerformanceFramework.objects.get(name=perf_datum['framework']['name'])
     for suite in perf_datum['suites']:
-        subtest_signatures = []
+        subtest_properties = []
+        summary_signature_hash = None
         for subtest in suite['subtests']:
-            subtest_properties = {
+            subtest_property = {
```

Comment:
Let's rename this to `subtest_metadata`. `subtest_property` is a bit confusing given the naming of everything else.